README.md

```diff
@@ -1,2 +1,179 @@
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need `git` locally installed. Please got to [Getting Started - Installing Git guide](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) and follow the steps described for your system to install `git`.
+
+### Setup
+
+Clone this repository to your desired folder:
+```sh
+  cd my-folder
+  git clone git@github.com:luigirazum/mv-leaderboard.git
+```
+
+### Install
+
+Install this project with:
+```sh
+  cd mv-leaderboard
+  npm install
+```
+
+### Build
+
+To build the project, execute the following command:
+```sh
+  npm run build  
+```
+
+### Run
+
+To run tests, run the following command:
+```sh
+  npm start
+```
+
+### Deployment
+
+You can deploy this project using:
+```sh
+  npm deploy
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👨‍💻 Luis Zubia
+
+- GitHub: <a href="https://github.com/luigirazum" target="_blank" rel="noopener noreferrer">@luigirazum</a>
+- Twitter: <a href="https://twitter.com/LuigiRazum" target="_blank" rel="noopener noreferrer">@LuigiRazum</a>
+- LinkedIn: <a href="https://linkedin.com/in/luiszubia" target="_blank" rel="noopener noreferrer">Luis Raul Zubia Maciel</a>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add some badges to players according to their score.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+In order to improve this project, contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+I really enjoyed making this project, so, if you like it, I appreciate your support giving me a ⭐.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ❓ FAQ <a name="faq"></a>
+
+- **Why should you use this project?**
+
+  - Because you can realize what you can achieve using this amazing tool.
+
+- **Why did I make this project?**
+
+  - In order to start putting in practice the use of WebPack, JS ES6 modules and API's.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```

Comment:
**[Optional]**

- [ ] Kindly remove these guidlines to make your readme file look clean because these guidlines are just for you not for anyone else 🤔